qemu/tests/balloon_check.py

```diff
@@ -17,6 +17,7 @@ class BallooningTest(MemoryBaseTest):
 
     def __init__(self, test, params, env):
         self.test_round = 0
+        self.test = test
```

Comment:
I guess the superclass would initialize `self.test` so you don't have to perform it again.

Reply:
Yes, you are right. Updated!